libraries/AP_NavEKF2/AP_NavEKF2_PosVelFusion.cpp

```diff
@@ -399,6 +399,9 @@ void NavEKF2_core::SelectVelPosFusion()
     // Check for data at the fusion time horizon
     extNavDataToFuse = storedExtNav.recall(extNavDataDelayed, imuDataDelayed.time_ms);
     extNavVelToFuse = storedExtNavVel.recall(extNavVelDelayed, imuDataDelayed.time_ms);
+    if (extNavDataToFuse) {
```

Comment:
small typo I think, should be "if (extNavVelToFuse)" shouldn't it?

Reply:
Oops. I am sorry. Thank you